### PR TITLE
RavenDB-16082 - add and endpoint for connectivity check to api.ravendb.net

### PR DESCRIPTION
**Pull request by grisha-kotler**

### CODE REVIEW

File: src/Raven.Server/Web/Studio/LicenseHandler.cs [hunk 1]
```diff
@@ -134,5 +135,44 @@ public async Task RenewLicense()
                 context.Write(writer, renewLicense.ToJson());
             }
         }
+
+        [RavenAction("/license-server/connectivity", "GET", AuthorizationStatus.ValidUser)]
```
Comment (ayende) [inline src/Raven.Server/Web/Studio/LicenseHandler.cs]: should this also be a debug endpoint?

Comment (ppekrol) [reply src/Raven.Server/Web/Studio/LicenseHandler.cs]: We can add it to the debug package, but the timout needs to be small, 3-5 seconds max IMO.

Do we also want to have access to this during setup?

File: src/Raven.Server/Web/Studio/LicenseHandler.cs [hunk 2]
```diff
@@ -134,5 +135,44 @@ public async Task RenewLicense()
...
+        public async Task CheckConnectivityToLicenseServer()
+        {
+            var result = new ConnectivityToLicenseServer();
+
+            try
+            {
+                var response = await ApiHttpClient.Instance.GetAsync("/Subscription.svc");
```
Comment (ppekrol) [inline src/Raven.Server/Web/Studio/LicenseHandler.cs]: You do not define a timeout which means it will take up to 100 seconds (default)

Comment (ppekrol) [reply src/Raven.Server/Web/Studio/LicenseHandler.cs]: I do not think that `/Subscription.svc` is an appropriate endpoint

First of all this is 3.5 endpoint, second this endpoint is not a lightweight endpoint. There should be a dedicated one e.g. `/alive` or sth like that

Comment (grisha-kotler) [reply src/Raven.Server/Web/Studio/LicenseHandler.cs]: The `/Subscription.svc` endpoint is lightweight (`GET` request) and was added for v3.5.
I'll add a new endpoint for this.

File: src/Raven.Server/Web/Studio/LicenseHandler.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Server/Web/Studio/LicenseHandler.cs]: done

File: src/Raven.Server/Web/Studio/LicenseHandler.cs (hunk 2 above)
Comment (grisha-kotler) [reply src/Raven.Server/Web/Studio/LicenseHandler.cs]: done

**Pull request merged**